src/components/Form/UploadDocumentForm.jsx

```diff
@@ -75,7 +75,7 @@ const UploadDocumentForm = () => {
           />
         </label>
         <div style={formRowStyle}>
-          <label htmlFor="upload-doc">Select document type to upload: </label>
+          <label htmlFor="upload-doc">Select Document Type</label>
           <DocumentSelection htmlId="upload-doc" />
```

Comment:
As far as I know, this should just be `<DocumentSelection id="upload-doc" />`

NOT `htmlId`

the reason why we have to add `html` to the `for` is because `for` is a reserved keyword in js.

I've never seen `htmlId`, and couldn't find anything on a search - so I don't believe it's required and may not be coupling these two together.

Reply:
You're correct, but the htmlId here is used as a prop to pass in the value for `id`, see `<select>` in DocumentSelection.jsx where htmlId used as a prop and used as the `id` attribute

Reply:
Yep!  deleting comment